MDCDecorator/MDCDecoratorEnd.yml

```diff
@@ -1,11 +1,10 @@
 steps:

```

Comment:
Should we change the directory/extension name to be DfD specific (vs. MDC)?

Reply:
Good point, let me rename these